### PR TITLE
Update the data_version_hanlder

### PR DESCRIPTION
**Pull request by kamilritz**
as discussed

### CODE REVIEW

File: src/ecl_ekf_analysis/log_processing/data_version_handler.py [hunk 1]
```diff
@@ -19,119 +20,171 @@ def get_output_tracking_error_message(ulog: ULog) -> str:
...
+            if elem.name == "ekf2_innovations":
+                return "ekf2_innovations"
+            if elem.name == "estimator_innovations":
+                return "estimator_innovations"
+    if topic == 'innovation_variance':
+        for elem in  ulog.data_list:
+            if elem.name == "ekf2_innovations":
+                return "ekf2_innovations"
+            if elem.name == "estimator_innovations":
+                return "estimator_innovations"
```
Comment (JohannesBrand) [inline src/ecl_ekf_analysis/log_processing/data_version_handler.py]: shouldn't this return `estimator_innovation_variances` ?

File: src/ecl_ekf_analysis/log_processing/data_version_handler.py [hunk 2]
```diff
@@ -19,119 +20,171 @@ def get_output_tracking_error_message(ulog: ULog) -> str:
...
+    if topic == 'innovation_test_ratio':
+        for elem in  ulog.data_list:
+            if elem.name == "ekf2_innovations":
+                return "ekf2_innovations"
+            if elem.name == "estimator_innovations":
+                return "estimator_innovations"
```
Comment (JohannesBrand) [inline src/ecl_ekf_analysis/log_processing/data_version_handler.py]: and this the test ratio? like it is now, every if seems to do exactly the same, so there must be something wrong here?

File: src/ecl_ekf_analysis/log_processing/data_version_handler.py [hunk 3]
```diff
@@ -157,3 +210,40 @@ def field_name_wo_brackets(field_name: str) -> str:
     if field_name.endswith(']'):
         return field_name[:-3]
     return field_name
+
+def get_innovation_message_and_field_name(
+        ulog: ULog, field_descriptor: str, topic: str = 'innovation') -> tuple([str, str]):
```
Comment (JohannesBrand) [inline src/ecl_ekf_analysis/log_processing/data_version_handler.py]: change `tuple([str, str])` to `Tuple[str, str]` and add `from typing import Tuple` at top.

**Review by JohannesBrand — CHANGES_REQUESTED** [review]
I think there is still a bug in `get_innovation_message` or does it work?

File: src/ecl_ekf_analysis/log_processing/data_version_handler.py (hunk 1 above)
Comment (kamilritz) [reply src/ecl_ekf_analysis/log_processing/data_version_handler.py]: Good catch

**Review by JohannesBrand — APPROVED** [review]
thank you for the update

**Pull request merged**